staNMF/staNMF.py

```diff
@@ -250,14 +249,12 @@ def runNMF(self, **kwargs):
             indexoutputstring = "selectedcolumns" + str(K) + ".csv"
             indexoutputpath = os.path.join(path, indexoutputstring)
 
-            guessDF = pd.DataFrame.from_dict(self.guessdict)
-            guessDF.columns = ["Replicate", "Columns Selected"]
-            guessDF.to_csv(indexoutputpath)
 
-            for m in sorted(self.guessdict):
-                indexoutputfile.write(str(m) + '\t' + str(self.guessdict[m]) +
-                                      '\n')
-            indexoutputfile.close()
+            with open(indexoutputpath, "w") as indexoutputfile:
+                for m in sorted(self.guessdict):
```

Comment:
I recommend always using a package to write delimited text, such as `csv`, rather than hardcoding it. However, since this PR didn't introduce the issue, this comment may be out of scope.